src/screens/Donate/Donate.tsx

```diff
@@ -1,111 +1,85 @@
+
   return (
-    <Fragment>
-      <EmbedWrapper>
-        <iframe
-          title="Donate to Covid Act Now on GiveButter"
-          src="https://givebutter.com/embed/c/covidactnow"
-          width="100%"
-          height="100%"
-          style={{ maxWidth: '601px' }}
-          name="givebutter"
-          frameBorder="0"
-          scrolling="auto"
-          seamless
-          data-allowPaymentRequest
-        ></iframe>
-        <script src="https://givebutter.com/js/widget.js"></script>
-      </EmbedWrapper>
-      <ContentWrapper container justify="center">
-        <GridItem item xs={12} sm={6}>
-          <ContentSection>
+    <Container>
+      {isMobile && <Intro />}
+      <GiveButterEmbed />
+      <ContentWrapper>
+        {!isMobile && (
+          <Fragment>
             <Icon icon={loveLetter} />
-            <SectionHeader>How your donation helps</SectionHeader>
-            <BodyCopy>
-              We’re a tiny team working relentlessly to provide free, accessible
-              data to policy makers and the public. We need financial support to
-              pay for our infustructure costs, our small full-time staff, and
-              continue launching new features.
-            </BodyCopy>
-          </ContentSection>
-          <ContentSection>
-            <Icon icon={testTube} />
-            <SectionHeader>Our theory of change</SectionHeader>
-            <BodyCopy>
-              By providing clearer, more accessible, locally-relevant and
-              science-based guidance that the general public and decision-makers
-              can understand, rally behind and communicate, we can advance a
-              more unified and more effective response to COVID in the U.S.
-            </BodyCopy>
-            <Link to="/about">Learn more about us</Link>
-          </ContentSection>
-        </GridItem>
-        <GridItem item xs={12} sm={6}>
-          <ContentSection>
-            <Icon icon={deliveryTruck} />
-            <SectionHeader>Our roadmap</SectionHeader>
-            {roadmapListContent.map((section, i) => {
-              return (
-                <Fragment>
-                  <ListHeader>{section.header}</ListHeader>
-                  <ListSection key={section.header}>
-                    {section.listItems.map((listItem, i) => {
-                      return <li key={`list-item-${i}`}>{listItem}</li>;
-                    })}
-                  </ListSection>
-                </Fragment>
-              );
-            })}
-          </ContentSection>
-        </GridItem>
-        <PartnersSection />
+            <Intro />
+          </Fragment>
+        )}
+        {roadmapListContent.map((item: any, i: number) => (
+          <Fragment>
+            <SectionHeader>{item.sectionHeader}</SectionHeader>
+            <BodyCopy>{item.copy}</BodyCopy>
+            <ul>
+              {item.listItems.map((item: any, i: number) => (
+                <BodyCopy as="li">{item}</BodyCopy>
```

Comment:
Remember to add a `key` when you map over items so React can do updates more efficiently


```jsx
<BodyCopy as="li" key={i}>{item}</BodyCopy>
```